cli/tests/error_syntax.js.out

```diff
@@ -1,4 +1 @@
-error: Uncaught SyntaxError: Unexpected identifier
-(the following is a syntax error  ^^ ! )
-     ~~~~~~~~~
-    at [WILDCARD]tests/error_syntax.js:3:6
+error: Expected Comma, got Some(Word(following)) at [WILDCARD]tests/error_syntax.js:3:5
```

Comment:
I think we need to improve these over time, as `got Some(Word(follow))` is pretty inaccessible to developers.  Ultimately we want go grab the source line and highlight the span, and put some sort of match in to humanise the errors.